src/views/ErrorBoundary/ErrorPage.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect } from 'react';
+import { useTracking } from 'react-tracking';
+
+import { useStateValue } from '../../store/store';
+import { i18n } from '../../utils';
+
+const ErrorPage = () => {
+	const [{ canonicalHost, language }] = useStateValue();
+	const tracking = useTracking();
+
+	useEffect(() => {
+		const pageTitle = 'Errors Occurred';
+		tracking.trackEvent({
+			event: 'SitewideSearchApp:Load:Error',
```

Comment:
Fix this please and make it `DrugDictionaryApp`

Reply:
Arghhh! Done. I'll update the Playground to have this passed in as props.